src/EditorFeatures/Core/ContentTypeNames.cs

```diff
@@ -4,13 +4,13 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     internal static class ContentTypeNames
```

Comment:
I will add [obsolete] in different PR. touching too many files.

Reply:
ah. that's good idea. let me do that

Reply:
so, disadvantage of that is I can't change namespace. but I guess that's okay?

Reply:
➡️ This now uses a type forwarder. Further discussion on this matter, if any, should be added to #30868 where this change was moved.